imgui_ros/TexturedShape[] shapes
---
string message  # ""
bool success  # False
